python/tank/deploy/tank_commands/setup_project_params.py

```diff
@@ -1194,7 +1197,32 @@ def get_uri(self):
         :returns: string
         """
         return self._config_uri
-        
+
+    def is_local_configuration(self):
```

Comment:
How about configured_project. existing_project sounds like there might not be a configuration yet.